pkg/database/migrations/3_dvd.up.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE `dvds` (
+  `id` integer not null primary key autoincrement,
+  `name` varchar(255),
```

Comment:
Make this `not null`, and the golang model can be changed from `NullString` to `string`, and the resolver for name is no longer necessary.

Reply:
I will change it.